docs/content/darkLight.js

```diff
@@ -0,0 +1,84 @@
+import {LitElement, html, css} from 'https://cdn.jsdelivr.net/gh/lit/dist@2/core/lit-core.min.js';
+import 'https://code.iconify.design/iconify-icon/1.0.7/iconify-icon.min.js';
+
+export class DarkLightToggle extends LitElement {
+    static properties = {
+        _theme: 'light'
+    }
+
+    constructor() {
+        super();
+        this.theme = localStorage.getItem("theme");
+        if (this.theme === "dark") {
+            document.documentElement.setAttribute("data-bs-theme", "dark");
+            this._theme = "dark";
+        }
+    }
+
+    toggleTheme() {
```

Comment:
It's worth considering using [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) so we use whatever someone's systems defaults are.

Reply:
<img width="128" alt="image" src="https://github.com/fsprojects/FSharp.Formatting/assets/1490044/c9248016-d370-4e7f-b616-abd1970efa04">


I know it's not as tidy as a toggle button but I think respecting the user's system theme is more important. Some example code: [code](https://github.com/TheAngryByrd/IcedTasks/blob/cdb90c260daabb499f8a0988046238dc933061a3/docsSrc/content/theme-toggle.js#L17)